readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -128,9 +129,9 @@
       <hr/>
       {% block footer %}
         <small>
-          <span>Hosted by <a href="https://readthedocs.org">Read the Docs</a></span>
+          <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">{% trans "Read the Docs" %}</a></span>
```

Comment:
Read the Docs is the company name, we shouldn't translate that, also, probably you want to use a blocktrans like https://github.com/stsewd/readthedocs.org/blob/8c2f8e77731dfc53149f3baa791abe080a9809a3/readthedocs/templates/projects/project_translations.html#L23-L27

Reply:
Right, ok.

The difference
- blocktrans: 
https://docs.djangoproject.com/en/dev/topics/i18n/translation/#std:templatetag-blocktrans
- trans: https://docs.djangoproject.com/en/dev/topics/i18n/translation/#std:templatetag-trans

I think the existing tagging is best. If there is anything other than `href` or text, we can use blocktrans. Useful summary: https://stackoverflow.com/a/17621832.